### PR TITLE
semicolon on comment issue solved. Issues #17635

### PR DESCRIPTION
**Pull request by hameddanesh**
semicolon on comment issue solved. Issues #17635

### CODE REVIEW

**Comment by hameddanesh** [comment]
unnecessary changes on previous commit undone.

**Comment by hameddanesh** [comment]
@williamdes 
unnecessary changes on previous commit undone

File: libraries/classes/Controllers/Import/ImportController.php [hunk 1]
```diff
@@ -204,6 +204,10 @@ public function __invoke(ServerRequest $request): void
             $GLOBALS['format'] = 'sql';
             $_SESSION['sql_from_query_box'] = true;
 
+            if (strpos($GLOBALS['import_text'], " -- ") != -1 && substr($GLOBALS['import_text'], -1) == ';') {
```
Comment (kamil-tekiela) [inline libraries/classes/Controllers/Import/ImportController.php:207]: `strpos` never returns `-1` and the condition should use identical operator `===`. You can also use `str_ends_with()`. 

I am not sure that this is the correct fix for this issue though.

Comment (hameddanesh) [reply libraries/classes/Controllers/Import/ImportController.php:207]: thank you for your suggestions but actually this solution works, you can compile and see it for yourself.
but also i commit that it's not the best solution to the problem at hand (i couldn't find were semicolon gets concatnate to query string so i fixed bug here, if you can guide be to the correct function i will be grateful)
and about strpos() it returns 0 to maxInt if needle is found and -1 (or false as you may mean it ) if it didn't.
anyway, thank you for your honest review, ill try to get better. 🌹

**Comment by williamdes** [comment]
Hi @hameddanesh 
Please do not forget to create a pull-request from another branch than `master`, the best is to create a branch from `master` branch and not use `master` directly. You will find some usefull information on: https://github.com/phpmyadmin/phpmyadmin/wiki/GIT_MANUAL

**Pull request closed without merging**